metadata.rb

```diff
@@ -27,5 +27,17 @@
   # so from Chef >= 14 chef_handler cookbook is deprecated.
   depends 'chef_handler', '>= 1.2'
 end
-depends    'apt' # Use '< 6.0.0' with Chef < 12.9
-depends    'yum', '>= 3.0' # Use '< 5.0' with Chef < 12.14
+
+# Use '< 6.0.0' with Chef < 12.9
+if Chef::VERSION < Gem::Version.new(12.9)
+  depends 'apt', '< 6.0'
+else
+  depends 'apt'
+end
+
+# Use '< 5.0' with Chef < 12.14
+if Chef::VERSION < Gem::Version.new(12.14)
+  depends 'yum', '< 5.0'
```

Comment:
So this should be both >= 3.0 and < 5.0, right?

Reply:
Right ! Nice catch, fixed in [166fd1f](https://github.com/DataDog/chef-datadog/pull/874/commits/166fd1f0d6e36b6d1b5b13681b2638eb0039b760).